packages/extensionmanager/src/companions.tsx

```diff
@@ -96,6 +96,37 @@ export interface IJupyterLabPackageData {
   };
 }
 
+function getInstallCommands(info: IInstallInfo) {
+  let commands = Array<string>();
+  if (
```

Comment:
This is a minor thing, but what do you think about using [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/overview.html#optional-chaining) here to make this a bit more succinct? And for below?

Reply:
Thx Saul, I have implemented a chaining which makes the code lighter.